oauth2_client/tests/test_logging_cmd.py

```diff
@@ -41,8 +41,8 @@ def setUpClass(cls):
     @classmethod
     def tearDownClass(cls):
         super(TestLoggingCommand, cls).tearDownClass()
-        # Restore test logging to ERROR only. We want clean output.
-        logging.disable(logging.WARNING)
+        # Restore test logging to ERROR and WARNING only. We want clean output.
```

Comment:
I very much approve of this. We need clarity on what constitutes a "failure" and to classify our messages accordingly.